.github/workflows/build.yml

```
@@ -48,14 +48,23 @@ jobs:
     - name: Install dependencies
       run: python -m pip install -U "jupyterlab>=4.0.0,<5"
 
+    - name: Restore Playwright browsers cache
+      uses: actions/cache@v3
+      id: playwright-cache
+      with:
+        path: /home/runner/.cache/ms-playwright
+        key: playwright-browsers-${{ runner.os }}-v1
+        restore-keys: |
+          playwright-browsers-${{ runner.os }}-
+      
```

```suggestion
    - name: Restore Playwright browsers cache
      uses: actions/cache@v4
      with:
        path: /home/runner/.cache/ms-playwright
        key: playwright-browsers-${{ runner.os }}-v1
        restore-keys: |
          playwright-browsers-${{ runner.os }}-
      
```

actions/cache@v3 is now defunct after February 1st, 2025. Could we use v4 instead?

See https://github.com/actions/toolkit/blob/683703c1149439530dcee7b8c5dbbfeec4104368/packages/cache/README.md#%EF%B8%8F-important-changes

Also, we could perhaps drop the `playwright-cache` ID as we're not using it anywhere below.